frontend/src/stores/filterState/FilterState.ts

```diff
@@ -36,14 +36,167 @@ export interface FilterLineState {
 /**
  * State of a filter
  */
-export interface FilterState {
+export class FilterState {
+  constructor(
+    private _edges: FilterLineState[],
+    private _nodes: FilterLineState[]
+  ) {}
+
+  get edges(): FilterLineState[] {
+    // eslint-disable-next-line no-underscore-dangle
+    return this._edges;
+  }
+
+  get nodes(): FilterLineState[] {
+    // eslint-disable-next-line no-underscore-dangle
+    return this._nodes;
+  }
+
+  /**
+   * Gets the isActive value of a {@link FilterLineState}.
+   * @param filterLineType - filter line which is searched for
+   * @param entity - specifies whether this entity is a node or an edge
+   */
+  public getFilterLineIsActive(
+    filterLineType: string,
```

Comment:
What is a filter-line type? Is this the entity (node/edge) type that the filter-line filters? If yes, specify this in the API doc. However it is, the naming is not very precise.